app/assets/javascripts/controllers/schedule/schedule_form_controller.js

```diff
@@ -59,7 +59,7 @@ ManageIQ.angular.app.controller('scheduleFormController', ['$http', '$scope', 's
         $scope.scheduleModel.name         = data.schedule_name;
         $scope.scheduleModel.timer_typ    = data.schedule_timer_type;
         $scope.scheduleModel.timer_value  = data.schedule_timer_value;
-        $scope.scheduleModel.start_date   = data.schedule_start_date;
+        $scope.scheduleModel.start_date   = moment.utc(data.schedule_start_date, 'MM/DD/YYYY').toDate()
```

Comment:
Currently, if `data.schedule_time_zone` is not UTC, then nothing will happen really. The way things are right now,
the date (year, month & day) are contained in one JS variable, time and hour in another variables and time zone
in another variable. `data.start_date` does not contain all the info we need (although it should).

Moreover, right now, these model values are not being submitted anywhere.